live-examples/js-examples/date-gettimezoneoffset.html

```diff
@@ -0,0 +1,12 @@
+<pre>
+<code id="static-js">var date1 = new Date('Tue, August 19, 1975 23:15:30 GMT+07:00');
+var date2 = new Date('Tue, August 19, 1975 23:15:30 GMT-02:00');
+
+console.log(date1.getTimezoneOffset());
+// expected output: your local timezone offset in minutes
+// (eg -120). NOT the timezone offset of the date object.
+
+console.log(date1.getTimezoneOffset === date2.getTimezoneOffset);
```

Comment:
Should this be:

`console.log(date1.getTimezoneOffset() === date2.getTimezoneOffset());`

Reply:
Yup.